From: Ben Hutchings <ben@decadent.org.uk>
Date: Tue, 13 Aug 2019 15:51:45 +0100
Subject: macvtap, tun: Avoid ABI change in 3.16.72
Forwarded: not-needed

Commit 13e380ca38fe "Revert "drivers/net, ipv6: Select IPv6 fragment
idents for virtio UFO packets"" deleted a #include from these drivers,
which changed the hash for some exported symbols.  Add it back.

---
--- a/drivers/net/macvtap.c
+++ b/drivers/net/macvtap.c
@@ -16,6 +16,7 @@
 #include <linux/idr.h>
 #include <linux/fs.h>
 
+#include <net/ipv6.h>
 #include <net/net_namespace.h>
 #include <net/rtnetlink.h>
 #include <net/sock.h>
--- a/drivers/net/tun.c
+++ b/drivers/net/tun.c
@@ -65,6 +65,7 @@
 #include <linux/nsproxy.h>
 #include <linux/virtio_net.h>
 #include <linux/rcupdate.h>
+#include <net/ipv6.h>
 #include <net/net_namespace.h>
 #include <net/netns/generic.h>
 #include <net/rtnetlink.h>
